### PR TITLE
feat: Support Object value type in Reading

### LINKED ISSUE #1024: Support object value types in Reading
# 🚀 Feature Request

### Description
 A clear and concise description of the problem or missing capability...
Since the core-data will support Object reading, the SDK should handle the object value before sending the event

### Describe the solution you'd like
 If you have a solution in mind, please describe it.
Add code to handle the object reading for the read and write command.

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

### PR DESCRIPTION
**Pull request by weichou1229**
Closes #1024

Signed-off-by: bruce <weichou1229@gmail.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features) 
   Added doc description to swagger API https://github.com/edgexfoundry/edgex-go/pull/3742

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-c/blob/master/.github/CONTRIBUTING.md

## What is the new behavior?
Handle the object value type for the event reading

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by weichou1229** [comment]
This PR should hold until https://github.com/edgexfoundry/go-mod-core-contracts/pull/666 merged.

File: pkg/models/commandvalue.go [hunk 1]
```diff
@@ -429,6 +429,16 @@ func (cv *CommandValue) BinaryValue() ([]byte, error) {
 	return value, nil
 }
 
+// ObjectValue returns the value in object data type, and returns error if the Type is not Object.
+func (cv *CommandValue) ObjectValue() (interface{}, error) {
+	var value float64
```
Comment (lenny-intel) [inline pkg/models/commandvalue.go:434]: Why using `float64` here and not `interface{}`?
I see that it is a noop for the error, but it makes the code confusing.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Comment by lenny-intel** [comment]
@weichou1229 Please reference the Docs PR for this enhancement in the PR.

File: pkg/models/commandvalue.go (hunk 1 above)
Comment (weichou1229) [reply pkg/models/commandvalue.go]: Sorry, it's redundant, removed.

**Comment by weichou1229** [comment]
> @weichou1229 Please reference the Docs PR for this enhancement in the PR.

Added the link.

**Comment by weichou1229** [comment]
Upgrade the core-contracts lib from v2.0.1-dev.18  to v2.0.1-dev.19

**Pull request merged**